### PR TITLE
Issue93 sidebar styling

### PR DESCRIPTION
**Pull request by tazo90**
Code was written for small screens first, then for the large.

Summary
- [x] navbar collapse animation
- [x] some styles for sidebar put back
- [x] smooth animation while toggle sidebar items
- [x] fixed page container

Tested on: `Firefox`, `Opera`, `Chrome`
Need to test on: `Safari`

### CODE REVIEW

**Comment by lsbardel** [comment]
can you make sure it is up to date with dev please?

File: lux/extensions/admin/sidebar.py [hunk 1]
```diff
@@ -26,6 +26,9 @@ def add_css(all):
     sidebar.toggle.border = '1px solid #ddd'
     sidebar.info.p.color = '#ccc'
     sidebar.info.background = '#425466'
+    sidebar.menu.max_height = px(500)
```
Comment (lsbardel) [inline lux/extensions/admin/sidebar.py]: what is this for? I can see that is used for transitions, but why 500px?

Comment (tazo90) [inline lux/extensions/admin/sidebar.py]: We need to explicitly set `max-height` to apply smooth transition, should I change 500px to 999px?

**Comment by tazo90** [comment]
Yes, the code was updated with the current dev.

**Comment by lsbardel** [comment]
there are still few issues.

* The sidebar initial state is open and than it collapses, you can see that by refreshing the page and see it collapsing.
* The navbar when minimised has two issues:
   * the Admin on the left and the toggle on the right are 50px in height, they should be 60px, the `small_height` variable. You can see that in the picture.
   * The lower border is double somehow and when expanded the lower border is missing and the upper border (lower border when closed) is double.

screen shot 2015-08-07 at 11 05 05

**Comment by tazo90** [comment]
Thanks for review, I'll do the fixes soon.

**Comment by lsbardel** [comment]
No this does not work properly.
The navbar does not work as it should.

**Comment by tazo90** [comment]
The issue it is because we use `navbar-fixed-top navbar-static-top` inside navbar.
The style `position:fixed` is applied to the navbar and it's was removed from the flow of the document, it is because collapsed menu not pushing content down when expanded.

**Comment by tazo90** [comment]
This can be close, because fixes was separated.

Navbar fix:
- [https://github.com/quantmind/lux/pull/107](https://github.com/quantmind/lux/pull/107)

Sidebar fix:
- [https://github.com/tazo90/lux/tree/sidebar-styling](https://github.com/tazo90/lux/tree/sidebar-styling)

**Pull request closed without merging**